### PR TITLE
fix(README): Document prerequisites and common variables for deployment

### PR DESCRIPTION
**Pull request by m-czernek**
## What does this PR change?

In this PR, I am documenting:

* Libvirt prerequisites: this seemed reasonable to me since the section also contains description of installing the `terraform-provider-libvirt` package
* Variables: a newcomer to Uyuni probably won't know what `CC` stands for. Additionally, the `images` section might be helpful

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,23 @@
...
 
-openSUSE and SUSE Linux Enterprise Server:
+### Prerequisites
+
+If you use the `Libvirt` provider, install and enable `libvirt` before you attempt to run the terraform deployment.
+The `virt-manager` package is recommended because it configures default resources that the terraform deployment uses, e.g. the `default` virtual network.
+
+```bash
+# Download and install libvirt and virt-manager, for example:
+sudo zypper install libvirt virt-manager
```
Comment (meaksh) [inline README.md:21]: It seems this line is SUSE/openSUSE specific, but we are in the general prerequisite section (which could apply also to Debian/Ubuntu).

Can we maybe make this more generic or maybe include this part in the "openSUSE / SUSE" section? What do you think?

**Review by meaksh — COMMENTED** [review]
Thanks a lot for these improvements! :tada:

I just made some suggestions that maybe you could consider. Thanks in advance!

**Comment by agraul** [comment]
There is also https://github.com/uyuni-project/sumaform/pull/769 which covers some of the same changes. Maybe we should also take a look at that one.

File: README.md (hunk 1 above)
Comment (m-czernek) [reply README.md]: Changed to:

a) move to a better (IMHO) location
b) include Debian/Ubuntu code example

**Comment by m-czernek** [comment]
@agraul I think this PR supersedes changes in #769 , doesn't it? IMO the `cc_*` variables are necessary for any backend, so it probably doesn't make sense to document them in the libvirt module. Perhaps we can delete the current `cc` docs in https://github.com/uyuni-project/sumaform/blob/master/backend_modules/libvirt/README.md? 

```
complete the cc_password variable in the base module
```

**Pull request merged**